site/faq.md

```diff
@@ -55,3 +55,24 @@ OSS Scope has no user concept, this is only available in Weave Cloud. To limit t
   -app.basicAuth.username string
         Username for basic authentication (default "admin")
   ```
+
+## ARM Support
+
+- It required patches, @adivyoseph (on [#scope](https://weave-community.slack.com/messages/scope/)) had done some work on this.
+- [#2110](https://github.com/weaveworks/scope/issues/2110) says that scope's CI builds ARM32 (but not ARM64) for test-builds at least.
+- @errordeveloper says: It should be easy to add arm64 in CI, You can try and enable builds in ci on a branch.. In theory, you just need to build for `GOARCH=arm64`.
+
+## Data Storage
+
+OSS Scope reports aren't persistent and the probe keeps the last 15 seconds of metrics in memory.
```

Comment:
The OSS codebase supports persistent reports as well, but it doesn't help you setting up the infrastructure required to do so.

Reply:
Yeah that's a good point, I will update the docs!